### ISSUE TITLE
Crashes when you don't assign the result to a variable

### ISSUE THREAD

**Issue by lucasbento** [hamlim/inline-mdx.macro#12]
So, I'm back 👋😅

I'm working on getting the [React Native Upgrade Helper](https://github.com/react-native-community/upgrade-helper) to include this library and I'm running into the following crash:

![image](https://user-images.githubusercontent.com/6207220/60704820-cf1dcb80-9f05-11e9-818f-4a5b697ee7dc.png)

Basically this is because I'm using it [like this](https://github.com/react-native-community/upgrade-helper/blob/ec7e61950d20a9e9b8f1be9266afe28dc69fbb70/src/releases/0.60.js#L25-L31), if I changed it to be like the following then it works:

```jsx
const MyComment = inline`
  ~~markdown~~
`

// ...

{
  fileName: 'ios/Podfile',
  lineNumber: 4,
  lineChangeType: 'add',
  content: MyComment
}
```

The way that I'm parsing the files inside of the release is:

1. I import all of them in the `index.js`: https://github.com/react-native-community/upgrade-helper/blob/ec7e61950d/src/releases/index.js#L1-L6 
1. I walk through the release files: https://github.com/react-native-community/upgrade-helper/blob/ec7e61950d/src/components/common/Diff/DiffComment.js#L61
1. Then show the content with `inline-mdx.macro`: https://github.com/react-native-community/upgrade-helper/blob/ec7e61950d/src/components/common/Diff/DiffComment.js#L61

If this is too much to follow I can definitely create a smaller project to reproduce it better.

**Comment by hamlim** [comment]
Cool, it seems like the logic here: https://github.com/hamlim/inline-mdx.macro/pull/11/files#diff-902b62f6db1e2896b465a0d02ef86479R32

```js
const funcName = reference.parentPath.parent.id.name;
```

Should handle cases where the assignment isn’t to a variable, right now it only handles 

```js
const Something = inline``;
```

but it should also handle:

```js
({
  identifier: inline``
})
```

I hope I can find some time to look into this later today or tomorrow

**Comment by hamlim** [comment]
Ok I published 1.0.0-alpha.3 with a fix for this bug, @lucasbento could you test that version and see if it resolves the issue?

**Comment by lucasbento** [comment]
Works really well, thank you @hamlim.

I noticed that it doesn't work sometimes based on the indentation, like this it works:

```
      content: inline`
~~strikethrough~~
      `
```

But like this it doesn't:

```
      content: inline`
        ~~strikethrough~~
      `
```

Is this intended? If not, should I create a new issue for it?

**Comment by hamlim** [comment]
Hey sorry for the delay in getting back to this issue! I'm going to try and take a look at that formatting bug today or tomorrow!

**Comment by hamlim** [comment]
Ok, I published v1.0.0-alpha.4 that should resolve any issues with nested template strings like you shared above.

Feel free to test it out and let me know how it goes!

If that works I will merge in the V1 PR and cut an official release!

**Comment by lucasbento** [comment]
@hamlim: that works perfectly!

![chef kiss](https://media1.tenor.com/images/a7031e8ce4a66c2aaafd87aca298290a/tenor.gif)

I believe it's good to ship!

Thank you so much for your work.
